### PR TITLE
Add autoflake to remove unused imports in cloud modules

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

This PR adds a task to the module_openapi_cloud role to remove unused imports from the generated module.

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: roles/module_openapi_cloud/tasks/main.yaml [hunk 1]
```diff
@@ -31,6 +31,12 @@
       - ( plugin['action'] == 'generate_ignore_files' ) or ( plugin['action'] == 'generate_all' )
       - plugin['name'] == 'vmware_rest'
 
+- name: Use autoflake to remove unused vars
+  ansible.builtin.command: "autoflake --in-place --remove-unused-variables -r {{ collection['path'] }}"
```
Comment (mariolenz) [inline roles/module_openapi_cloud/tasks/main.yaml:35]:
```suggestion
- name: Use autoflake to remove unused imports
  ansible.builtin.command: "autoflake --in-place --remove-all-unused-imports  -r {{ collection['path'] }}"
```

We don't have a problem with unused vars, at least not in vmware.vmware_rest ;-)

Comment (GomathiselviS) [reply roles/module_openapi_cloud/tasks/main.yaml:35]: My bad! will fix this.

**Review by mariolenz — APPROVED** [review]
LGTM

I've tested in ansible-collections/vmware.vmware_rest#438 but for other reasons I've mentioned elsewhere. The unused imports seem to have gone :-)

Thanks @GomathiselviS!

**Review by abikouo — APPROVED** [review]

**Comment by mariolenz** [comment]
Thanks @GomathiselviS and @alinabuzachis!

**Pull request merged**